Update authenticator

eepyfemboi opened the pull request:
Made authenticator automatic. No more copy/pasting links and codes and urls.
requires selenium to be installed*

eepyfemboi committed:
Update authentication.py

eepyfemboi committed:
Update authentication.py

eepyfemboi committed:
Update requirements.txt

eepyfemboi committed:
Update authentication.py

TargetedEntropy commented:
Hello, currently getting this error in testing:
``` 
 File "C:\Users\user\Desktop\git\coco-pycraft\minecraft\authentication.py", line 364, in GetoAuth20
    time.sleep(1)
NameError: name 'time' is not defined
```

eepyfemboi committed:
Update authentication.py

added tiem import statement

eepyfemboi committed:
Add files via upload

copy/pasted chromedrivers for compatibility purposes. will add an os check in setup.py soon^tm...

eepyfemboi committed:
Update setup.py

eepyfemboi commented:
Should be mostly fixed. Haven't fully tested it yet on VMs, but the os check in setup.py should (theoretically) install the correct webdriver.
Again, further testing is needed.

TargetedEntropy commented:
Looks great, thank you!
